### DISCUSSION TITLE
pretix without separate subdomain

### DISCUSSION THREAD

**Discussion by Caseit18** [pretix/pretix#4650] [Q&A] [answered]
Our org makes requesting additional subdomains (host names, etc) for our external domain.  I was hoping I'd be able to do something as simple as https://www.ourdomain.org/pretix  Is there documentation or has someone made this work elsewhere.

I'm struggling to find documentation on how to do that and just winging it with the pretix.cfg & the proxy configuration get me about 15% of the way there. I can get the landing page to go, but additional resources act funky. So the landing page wants me to go to the control by going to https://www.ourdomain.org/control  and if i manually going https://www.ourdomain.org/pretix/control it redirects me to: https://www.ourdomain.org/control/login?next=/control/ I can then fix that and do https://www.ourdomain.org/pretix/control/login?next=/control/ and that page loads, BUT through all of this, all of the resources (images, etc) are incorrect and I have a feeling my problems are just beginning.

The problem is _somewhat_ complicated because I can't use nginx (as there is an existing apache http server on this system). It's running and I don't believe this is the problem as data is being proxied back and forth.

Does pretix just only function if it's at the equivalent of root of a website?

**Comment by raphaelm** [comment, marked as answer]
Yes, pretix needs to be on it's own subdomain, running it under a subpath is not supported.

**Reply by Caseit18** [reply]
Thank you for the straight forward answer.
Rather than saying "will you please support this", I'd like to learn about the design of it. Was this one of those "what idiot would try hosting this as a subpath?" situations or "It is a real pain to fight with XYZ and make it play nice" decisions, or just "That never came up and isn't in the expected use case."

**Reply by raphaelm** [reply]
We supported this in the very beginning (many years ago) and noticed that support for it in third-party Django packages was quite bad, so we removed it to avoid unpredictable issues. I don't think we use the third-party packages that originally caused this any more, but we've now worked under the assumption of a root URL for so long that I don't think we'll go through the effort of changing it. (It would also create some weird questions around handling of secondary domains.)
